ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container-component.test.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import sinon from 'sinon';
-import { shallowWithContext } from '../../../../../test/lib/render-helpers';
-import { getGasFeeEstimatesAndStartPolling } from '../../../../store/actions';
+import configureMockStore from 'redux-mock-store';
+import {
+  shallowWithContext,
+  renderWithProvider,
+} from '../../../../../test/lib/render-helpers';
+import { updateWithAndStartPollingFor } from '../../../../store/actions';
 
 import PageContainer from '../../../ui/page-container';
 
 import { Tab } from '../../../ui/tabs';
 import GasModalPageContainer from './gas-modal-page-container.component';
 
 jest.mock('../../../../store/actions', () => ({
-  disconnectGasFeeEstimatePoller: jest.fn(),
-  getGasFeeEstimatesAndStartPolling: jest
+  stopPollingFor: jest.fn(),
+  updateWithAndStartPollingFor: jest
     .fn()
     .mockImplementation(() => Promise.resolve()),
   addPollingTokenToAppState: jest.fn(),
+  removePollingTokenFromAppState: jest.fn(),
 }));
 
```

Comment:
`'SLOW'` I think is not values value for `gasEstimateType`, but yep ok as a dummy value.